storm-core/src/jvm/backtype/storm/messaging/netty/Client.java

```diff
@@ -66,8 +67,14 @@
     private AtomicLong flushCheckTimer;
     private int flushCheckInterval;
     private ScheduledExecutorService scheduler;
+    
+    private CountDownLatch authenticatedSignal = null;
+
+    public CountDownLatch getAuthenticatedSignal() {
+		return authenticatedSignal;
+	}
 
-    @SuppressWarnings("rawtypes")
+	@SuppressWarnings("rawtypes")
```

Comment:
It seems the existing Java source uses an indentation of four spaces, but the new source uses eight.  It would be better I think to keep it the same as the existing source.